python/cuspatial/cuspatial/tests/test_geoseries.py

```diff
@@ -181,13 +183,23 @@ def test_interleaved_point(gs, polys):
             dtype="float64",
         ).reset_index(drop=True),
     )
+
+    polys_list = gs[gs.apply(lambda x: isinstance(x, (MultiPolygon, Polygon)))]
+    polys = list(
+        chain(polys_list.apply(get_coordinates))
+    )  # flatten multigeometries
+    coords_list = list(chain(*polys))  # flatten geometries
+    xy_interleaved = list(chain(*coords_list))  # flatten coordinates
+    x = xy_interleaved[::2]
+    y = xy_interleaved[1::2]
```

Comment:
Not something we need to fix in this PR, but it would be nice to abstract this into a helper.

Reply:
Agree, there is a lot of repetition.

Reply:
Working on this.

Reply:
Done in 5488ac6

Reply:
I assume you mean 1e961844296db93599561374c8cd27daa0d16fa9? This looks much cleaner. Thanks!